tests/unit/commands/list/resources/test_resources.py

```diff
@@ -0,0 +1,2 @@
+class TestResources:
```

Comment:
Let's remove this file for now. Other unit tests usually just have the logic housed in the `test_cli.py` which should be enough for now. We can break it off later if we need to.

Reply:
Removed